src/main/java/ch/njol/skript/expressions/ExprExplodedBlocks.java

```diff
@@ -0,0 +1,86 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import java.util.List;
+
+import org.bukkit.block.Block;
+import org.bukkit.event.Event;
+import org.bukkit.event.entity.EntityExplodeEvent;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.ScriptLoader;
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Events;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+
+@Name("Exploded Blocks")
+@Description("Get all the blocks that were destroyed in an explode event")
+@Examples({"on explode:",
+	"\tloop exploded blocks:",
+	"\t\tadd loop-block to {exploded::blocks::*}"})
+@Events("explode")
+@Since("INSERT VERSION")
+public class ExprExplodedBlocks extends SimpleExpression<Block> {
+
+	static {
+		Skript.registerExpression(ExprExplodedBlocks.class, Block.class, ExpressionType.COMBINED, "[the] (destroyed|exploded) blocks");
```

Comment:
is `destroyed` really a good idea? imagine someone tries
```
on break:
  broadcast "%destroyed blocks%"
```
wouldn't it be a bit annoying/confusing for them to get an error about it only working in explode events?

Reply:
I share this concern. I like the syntax, though; maybe just make it work with break events?

Reply:
Very understandable. I'm going to remove it, since `on break` already has `event-block` so I do not really see the need to be able to use `destroyed block` in a break event.

Reply:
Already it will error if used in a wrong event, there is nothing wrong with it...

Reply:
I would just remove the alias, people won't even use it due to the fact that skQuery uses `exploded` and people are used to that. And I also think people could get confused if someone ever happens to try it in a block break event.